content/algorithms/sdr/notation.md

```diff
@@ -560,18 +560,15 @@ The number of parent labels factored into each node label.
 `$\OctTreeDepth = \log_8(N_\nodes) = 10$`\
 The depth of a `$\BinTree$` and `$\OctTree$` respectively. The number of tree layers is the tree's depth `$+ 1 \thin$`. The Merkle hash arity of trees are 2 and 8 respectively.
 
-`$N_{\poreppartitions / \batch} = 10$`\
-`$N_{\postpartitions / \batch, \P, \thin \winning} \leq \len(\PostReplicas_{\P, \batch})$`\
-`$N_{\postpartitions / \batch, \P, \thin \window} \leq \len(\PostReplicas_{\P, \batch})$`\
-The number of partition proofs per PoRep, Winning PoSt, and Window PoSt proof batch. The number of PoSt partition proofs in a batch is specific to the size of the PoSt prover `$\P$`'s replica set `$\PostReplicas_{\P, \batch}$` at the time of batch proof generation.
-
 `$N_{\porepreplicas / k} = 1$`\
 `$N_{\postreplicas / k, \winning} = 1$`\
-`$N_{\postreplicas / k, \window} = 2349$`\
-The number of challenged replicas per PoRep, Winning PoSt, and Window PoSt partition proof.
+`$N_{\postreplicas / k, \window} = \big[ \hspace{1pt} \ell_\sector^\byte = 32\ \text{GiB} \mathbin{?} 2349 \mathbin{:} 2300 \hspace{1pt} \big]$`\
```

Comment:
I don't know if there's a difference between `\text{}` and `\mathbin{}` in these cases. If not, you could use something like (not tested) `\tern{32\ \text{GiB}}{2349}{2300}` instead.